src/Microsoft.Azure.WebJobs.Host/Bindings/DefaultResolutionPolicy.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Resolution policy for { } in  binding templates. 
+    /// The default policy is just a direct substitution for the binding data. 
+    /// Derived policies can enforce formatting / escaping when they do injection. 
+    /// </summary>
+    internal class DefaultResolutionPolicy<TAttribute> : IResolutionPolicy<TAttribute> where TAttribute : Attribute
+    {
+        public string TemplateBind(PropertyInfo propInfo, TAttribute resolvedAttribute, BindingTemplate template, IReadOnlyDictionary<string, object> bindingData)
+        {
+            if (bindingData == null)
+            {
+                return template?.Pattern;
+            }
```

Comment:
That means that if bindingData is null, my policy is never called. Maybe someone wants to do something special in that case. I didn't like that it restricted something that I'm trying to handle in a custom way.